lib/_tls_wrap.js

```diff
@@ -860,6 +861,33 @@ TLSSocket.prototype.renegotiate = function(options, callback) {
   return true;
 };
 
+TLSSocket.prototype.exportKeyingMaterial = function(length, label) {
+  if (typeof length !== 'number') {
+    throw new ERR_INVALID_ARG_TYPE(
+      'length',
+      'number',
+      length
+    );
+  }
+
+  assert(length > 0,
+         'length is not a positive number: ' + length);
```

Comment:
There are a bunch of validators in `lib/internal/validators.js` that you can use. `validateUint32(length, 'length', true)` is probably the one you want.